pip_faster.py

```diff
@@ -133,22 +133,25 @@ def find_requirement(self, req, upgrade):
             # if the version is pinned-down by a ==
             # first try to use any installed package that satisfies the req
             if req.satisfied_by:
-                logger.debug('Faster! pinned requirement already installed.')
+                logger.notify('Faster! pinned requirement already installed.')
                 raise BestVersionAlreadyInstalled
 
             # then try an optimistic search for a .whl file:
             link = optimistic_wheel_search(req, self.find_links)
             if link is None:
-                logger.notify('\033[1;93m[slower]\033[0m No wheel found locally for pinned requirement %s', req)
+                # The wheel will be built during prepare_files
+                logger.debug('No wheel found locally for pinned requirement %s', req)
             else:
-                logger.notify('\033[1;92m[fast]\033[0m Pinned wheel found, without hitting PyPI.')
+                logger.notify('Faster! Pinned wheel found, without hitting PyPI.')
                 return link
         else:
-            logger.notify('\033[1;91m[SLOW!]\033[0m Installing unpinned requirement %s', req)
+            # unpinned requirements aren't very notable. only show with -v
+            logger.info('slow: full search for unpinned requirement %s', req)
```

Comment:
I disagree with removing slow on unpinned.  That's the slowest of things and needs fix